### PR TITLE
Make it clear that we introduced a breaking change in 1.3.0

### PR DESCRIPTION
**Pull request by trustin**
.. so that users are informed better when they upgrade.

- Miscellaneous:
  - Fixed a broken GitHub username

### CODE REVIEW

File: site/src/pages/release-notes/1.3.0.mdx [hunk 1]
```diff
@@ -227,6 +227,15 @@ date: 2020-11-30
...
   <type://JettyService>. #3194
 - The route decorators are now evaluated in the reverse order they applied. #3160 #3166
+  - The purpose of this change is to make sure we provide a consistent experience when decorating a service.
+    For example, when we decorate a service like the following:
+    ```java
+    myService
+      .decorate(decoratorA)
+      .decorate(decoratorB);
+    ```
+    we expect `decoratorA` decorates `myService` and `decoratorB` decorates `decoratorA`.
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.3.0.mdx:237]: ?
```suggestion
    We expect `decoratorA` decorates `myService` and `decoratorB` decorates `decoratorA`.
```

**Review by ikhoon — APPROVED** [review]
Thanks! 🙇‍♂️

File: site/src/pages/release-notes/1.3.0.mdx (hunk 1 above)
Comment (trustin) [reply site/src/pages/release-notes/1.3.0.mdx:237]: It's actually a continuation of `the following` above.

File: site/src/pages/news/20200703-newsletter-1.mdx [hunk 2]
```diff
@@ -69,11 +69,11 @@ Sign up for our newsletters to keep up-to-date with new releases and useful tips
 <ThankYou
   message={() => <p>Since our last newsletter, we received awesome contributions from the following contributors:</p>}
   usernames={[
-    'adriancole',
     'amitvc',
     'andrey-tpt',
     'anuraaga',
     'cbornet',
+    'codefromthecrypt',
```
Comment (anuraaga) [inline site/src/pages/news/20200703-newsletter-1.mdx:76]: @codefromthecrypt you should have picked your new name to start with an `a` :p

**Review by anuraaga — APPROVED** [review]

**Review by minwoox — APPROVED** [review]
Thanks!

File: site/src/pages/news/20200703-newsletter-1.mdx (hunk 2 above)
Comment (ikhoon) [reply site/src/pages/news/20200703-newsletter-1.mdx:76]: 🤣

**Comment by lodewykk** [comment]
Thanks!

**Pull request merged**